versatile-data-kit: simplify release process (#951)
vmware/versatile-data-kit: CONTRIBUTING.md

To make a new public release, follow these steps:
- Create a new release from the Releases menu - this will require a new tag;
- Autogenerate the changelog (https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes), then sort the changes alphabetically; if all change names are prefixed with the component name, this will ensure changes are grouped by component. Sorting can be done using the `sort` utility available in MacOS and Linux distros; Clean up unnecessary commits (e.g automated commits like pre-commit.ci)
- Write a summary at the top of the release page. It should include sections:
  - Major features include
  - Backwards incompatible changes
  - Removal of features
- Review which components have received changes since the previous release - this should be apparent from the changelog;

> - Record the latest versions of the components included in teh release (from the auto generated notes) and list them in the Release description;

Reviewer: ```suggestion
- Record the latest versions of the components included in the release (from the auto generated notes) and list them in the Release description;
```

>     - You can see the current version (as of time of release) of python distribution in https://pypi.org/search/?q=

Reviewer: ```suggestion
    - You can see the current version (as of time of release) of a Python package in https://pypi.org/search/?q=
```

>     - You can see the current version (as of time of release) of control service in [helm repo here](https://gitlab.com/vmware-analytics/versatile-data-kit-helm-registry/-/packages)

Reviewer: ```suggestion
    - You can see the current version (as of time of release) of the Control Service in [the helm repo here](https://gitlab.com/vmware-analytics/versatile-data-kit-helm-registry/-/packages)
```
